adhocteam/Head-Start-TTADP
# Refactor activity report review pages

**Description of change**
 * Each page has more control over what shows up on it's review page
 * Topics and resources shows download links to the files that have been uploaded to the report

**How to test**

1) Pull down, fill out a report
2) Add files to the report
3) Look at the files on the review page. You may have to invert the logic in `frontend/src/pages/ActivityReport/Pages/Review/FileReviewItem.js` line 16 in order to see an actual download link (unless https://github.com/adhocteam/Head-Start-TTADP/pull/176 is merged). We only show the download link for approved files and files are never currently approved.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/247

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
